poll.py

```diff
@@ -26,14 +26,13 @@ def lambda_handler(event, context):
             Bucket = body['bucket']
             ConversionID = body['uploadID']
             Path = str(body['path'])
-            if not Path:
-                Path = 'NULL'
             Filename = body['fileName']
             RequestedFormats = body['sizeFormat']
-            if Path is not 'NULL':
-                VideoURL = "https://{}.s3.amazonaws.com/{}{}".format(Bucket, Path, Filename)
-            elif Path is 'NULL':
-                VideoURL = "https://{}.s3.amazonaws.com/{}".format(Bucket, Filename)
+            if Path is None:
+                Path = 'null'
```

Comment:
FYI this was put in place because you can't put empty strings in dynamo. kill me